ios/MeasureSDK/MeasureSDK/Config/MeasureConfig.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MeasureConfig.swift
+//  MeasureSDK
+//
+//  Created by Adwin Ross on 25/08/24.
+//
+
+import Foundation
+
+/// Configuration for the Measure SDK. See `MeasureConfig` for details.
+protocol MeasureConfigProtocol {
```

Comment:
Consider not adding `Protocol` suffix to protocols. Check the Swift [API design guideline](https://www.swift.org/documentation/api-design-guidelines/).

> * Protocols that describe what something is should read as nouns (e.g. Collection).
> * Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting)